packages/cli/src/internal/prompt/text.ts

```diff
@@ -58,19 +63,32 @@ function renderClearScreen(state: State, options: Options) {
 }
 
 function renderInput(nextState: State, options: Options, submitted: boolean) {
+  const text = getValue(nextState, options)
+
   const annotation = Option.match(nextState.error, {
-    onNone: () => submitted ? Ansi.white : Ansi.combine(Ansi.underlined, Ansi.cyanBright),
+    onNone: () =>
+      Match.value(submitted)
```

Comment:
I would remove the usage of `Match` here - it is just adding unnecessary overhead and we try to keep the internals of the libs as fast as possible.

Reply:
Sounds good!